mra6:

  joint_names:
    - J1
    - J2
    - J3
    - J4
    - J5
    - J6


  CAN_NODE_DEV: /dev/pcan32

